corehq/apps/sso/tests/test_tasks.py

```diff
@@ -133,3 +139,103 @@ def tearDown(self):
     def tearDownClass(cls):
         cls.account.delete()
         super().tearDownClass()
+
+
+class TestAutoDeactivationTask(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.account = generator.get_billing_account_for_idp()
+        cls.account.enterprise_admin_emails = ['test@vaultwax.com']
+        cls.account.save()
+        cls.domain = Domain.get_or_create_with_name("vaultwax-001", is_active=True)
```

Comment:
Add `cls.addClassCleanup(cls.domain.delete)`

Reply:
THANK YOU!!! Added in 602208719e9ee591431876a37c68bc37f357aee6